hook.php

```diff
@@ -344,11 +344,6 @@ function plugin_formcreator_hook_add_ticket(CommonDBTM $item) {
       return;
    }
 
-   if (isset($item->input['items_id'][PluginFormcreatorFormAnswer::getType()])) {
-      // the ticket is associated to a form answer
-      return;
-   }
```

Comment:
When the execution flow reaches this point, we are sure that the ticket is not addesd by Formcreator, then the ticket is not from a group of tickets genearated by a formanswer object. This early end of function causes inconsistencies.